en/02_Developer_Guides/05_Extending/06_Aspects.md

```diff
@@ -13,10 +13,6 @@ Aspect oriented programming is the idea that some logic abstractions can be appl
 > functions from the main program's business logic. It aims to increase modularity by allowing the separation of
 > cross-cutting concerns, forming a basis for aspect-oriented software development.
 
-[notice]
-[Wikipedia](http://en.wikipedia.org/wiki/Aspect-oriented_programming) provides a much more in-depth explanation.
-[/notice]
```

Comment:
Removed because it doesn't actually give any useful information, and was violating [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.